i3fyra is broken after your update today

```
i3fyra -a
[ERROR] @$ not a valid command
```

https://aur.archlinux.org/packages/i3ass

```
i3fyra --version
i3fyra - version: 1.25
updated: 2022-05-21 by budRich

```

--- a/README.md
+++ b/README.md
@@ -32,9 +32,15 @@ See the [last releasenote](docs/releasenotes/0next.md).
 ### usage
 
 The table below lists the included commands. The
-links go to the **wiki** page of each commands. 
+links go to the **[wiki]** page of each commands. 
 There is also a lot of videos on the budlabs [youtube channel],
 where i3ass is used and explained.
+
+In the **[wiki]** there are also two examples on how
+i3's config file can be configured to make use
+of most of **i3ass**.  
+(<https://github.com/budlabs/i3ass/wiki/i3-config-example>)  
+
 
 script | description
 |:-|:-|

--- a/docs/readme_about.md
+++ b/docs/readme_about.md
@@ -6,6 +6,12 @@ See the [last releasenote](docs/releasenotes/0next.md).
 ### usage
 
 The table below lists the included commands. The
-links go to the **wiki** page of each commands. 
+links go to the **[wiki]** page of each commands. 
 There is also a lot of videos on the budlabs [youtube channel],
 where i3ass is used and explained.
+
+In the **[wiki]** there are also two examples on how
+i3's config file can be configured to make use
+of most of **i3ass**.  
+(<https://github.com/budlabs/i3ass/wiki/i3-config-example>)  
+

--- a/docs/releasenotes/0next.md
+++ b/docs/releasenotes/0next.md
@@ -1,18 +1,9 @@
-## next
+### 2022.05.21
 
-### bashbud
+#### bashbud
 
-Many improvements to the build steps.
-Manpages are included so no need for manpage generator
-as dependency.
+fixed issue reported in #170, that made all scripts 
+broken (tried to start main() passing `@$` instead of `$@`).
 
-### [i3list]
-NEW KEY: `i3list[SUS]`  
-contains the number of containers matching all criteria
-
-### [i3run]
-When xdotool is used to rename a window class/instance
-the window gets a new `container_id`, and now that
-new id is reported instead of the old.
-
-don't use eval when executing commands. 
+Fixed issue that made all programs look like they 
+where updated on the installation data.

--- a/docs/releasenotes/2022.05.20.md
+++ b/docs/releasenotes/2022.05.20.md
@@ -1,0 +1,18 @@
+### 2022.05.20
+
+#### bashbud
+
+Many improvements to the build steps.
+Manpages are included so no need for manpage generator
+as dependency.
+
+#### [i3list]
+NEW KEY: `i3list[SUS]`  
+contains the number of containers matching all criteria
+
+#### [i3run]
+When xdotool is used to rename a window class/instance
+the window gets a new `container_id`, and now that
+new id is reported instead of the old.
+
+don't use eval when executing commands. 

--- a/share/Makefile
+++ b/share/Makefile
@@ -136,7 +136,7 @@ $(MONOLITH): $(CACHE_DIR)/print_version.sh $(NAME) $(CACHE_DIR)/print_help.sh $(
 			grep -vhE -e '^#!/' -e '#bashbud$$' $$f
 		done
 
-		printf '%s\n' '' 'main "@$$"'
+		printf '%s\n' '' 'main "$$@"'
 	} > $@
 	
 	chmod +x $@

--- a/share/Makefile
+++ b/share/Makefile
@@ -70,20 +70,23 @@ function_files := \
 # this is needed for _init.sh (BASE) to know it needs
 # to be rebuilt on this event.
 
-ifeq ($(wildcard $(CACHE_DIR)/got_func),)
-  $(shell mkdir -p $(CACHE_DIR)/options)
-  $(shell echo 0 > $(CACHE_DIR)/got_func)
+ifneq ($(wildcard $(CACHE_DIR)/got_func),)
+  ifneq ($(wildcard $(FUNCS_DIR)/*),)
+    ifneq ($(file < $(CACHE_DIR)/got_func), 1)
+      $(shell echo 1 > $(CACHE_DIR)/got_func)
+    endif
+  else
+    ifneq ($(file < $(CACHE_DIR)/got_func), 0)
+      $(shell echo 0 > $(CACHE_DIR)/got_func)
+    endif
+  endif
 endif
 
-ifneq ($(wildcard $(FUNCS_DIR)/*),)
-  ifneq ($(file < $(CACHE_DIR)/got_func), 1)
-    $(shell echo 1 > $(CACHE_DIR)/got_func)
-  endif
-else
-  ifneq ($(file < $(CACHE_DIR)/got_func), 0)
-    $(shell echo 0 > $(CACHE_DIR)/got_func)
-  endif
-endif
+$(CACHE_DIR)/got_func: | $(CACHE_DIR)/
+	@$(info making $@)
+	[[ -d $(FUNCS_DIR) ]] \
+		&& echo 1 > $@ \
+		|| echo 0 > $@
 
 clean:
 	rm -rf $(wildcard _*) $(CACHE_DIR) $(generated_functions)

--- a/share/Makefile
+++ b/share/Makefile
@@ -33,7 +33,7 @@ MANPAGE_OUT          = _$(MANPAGE)
 FUNC_STYLE          := "() {"
 
 ifneq ($(wildcard config.mak),)
-include config.mak
+  include config.mak
 endif
 
 manpage_section      = $(subst .,,$(suffix $(MANPAGE)))
@@ -41,19 +41,18 @@ function_createconf := $(FUNCS_DIR)/_createconf.sh
 function_awklib     := $(FUNCS_DIR)/_awklib.sh
 
 ifneq ($(wildcard $(CONF_DIR)/*),)
-include_createconf   = $(function_createconf)
-conf_dirs            = $(shell find $(CONF_DIR) -type d)
-conf_files           = $(shell find $(CONF_DIR) -type f)
+  include_createconf   = $(function_createconf)
+  conf_dirs            = $(shell find $(CONF_DIR) -type d)
+  conf_files           = $(shell find $(CONF_DIR) -type f)
 else
-$(shell rm -f $(function_createconf))
+  $(shell rm -f $(function_createconf))
 endif
 
 ifneq ($(wildcard $(AWK_DIR)/*),)
-
-include_awklib       = $(function_awklib)
-awk_files            = $(wildcard $(AWK_DIR)/*)
+  include_awklib       = $(function_awklib)
+  awk_files            = $(wildcard $(AWK_DIR)/*)
 else
-$(shell rm -f $(function_awklib))
+  $(shell rm -f $(function_awklib))
 endif
 
 option_docs          = $(wildcard $(DOCS_DIR)/options/*)
@@ -84,9 +83,8 @@ endif
 
 $(CACHE_DIR)/got_func: | $(CACHE_DIR)/
 	@$(info making $@)
-	[[ -d $(FUNCS_DIR) ]] \
-		&& echo 1 > $@ \
-		|| echo 0 > $@
+	[[ -d $${tmp:=$(FUNCS_DIR)} ]] && tmp=1 || tmp=0
+	echo $$tmp > $@
 
 clean:
 	rm -rf $(wildcard _*) $(CACHE_DIR) $(generated_functions)
@@ -195,65 +193,9 @@ $(CACHE_DIR)/help_table.txt: $(CACHE_DIR)/long_help.md
 		paste <(echo "$$frag") <(echo "$$desc") | tr -d '\t'
 	done > $@
 
-$(function_awklib): $(awk_files) | $(FUNCS_DIR)/
-	@$(info making $@)
-	{
-		printf '%s\n' \
-			'$(SHBANG)'                                                             \
-			''                                                                      \
-			'### _awklib() function is automatically generated'                     \
-			'### from makefile based on the content of the $(AWK_DIR)/ directory'   \
-			''
-
-		fstyle=$(FUNC_STYLE)
-		printf "_awklib$${fstyle}\n"
-		printf '%s\n' \
-			'[[ -d $$__dir ]] && { cat "$$__dir/$(AWK_DIR)/"* ; return ;} #bashbud' \
-			"cat << 'EOAWK'"
-		cat $(awk_files)
-		printf '%s\n' "EOAWK" '}'
-	} > $@
-	
-
-$(function_createconf): $(conf_files) | $(FUNCS_DIR)/
-	@$(info making $@)
-	{
-		printf '%s\n' \
-			'$(SHBANG)'                                                            \
-			''                                                                     \
-			'### _createconf() function is automatically generated'                \
-			'### from makefile based on the content of the $(CONF_DIR)/ directory' \
-			''
-
-
-		fstyle=$(FUNC_STYLE)
-		printf "_createconf$${fstyle}\n"
-
-		echo 'local trgdir="$$1"'
-
-		echo 'mkdir -p $(subst $(CONF_DIR),"$$trgdir",$(conf_dirs))'
-		for f in $(conf_files); do
-
-			echo ""
-			echo 'if [[ -d $$__dir ]]; then #bashbud'
-			echo "cat \"\$$__dir/$$f\" > \"$${f/$(subst /,\/,$(CONF_DIR))/\$$trgdir}\" #bashbud"
-			echo 'else #bashbud'
-			echo "cat << 'EOCONF' > \"$${f/$(subst /,\/,$(CONF_DIR))/\$$trgdir}\""
-			cat "$$f"
-			echo "EOCONF"
-			echo 'fi #bashbud'
-			[[ -x $$f ]] && echo "chmod +x \"$${f/$(subst /,\/,$(CONF_DIR))/\$$trgdir}\""
-		done
-
-		echo '}'
-	} > $@
-
 $(CACHE_DIR)/:
 	@$(info creating $(CACHE_DIR)/ dir)
 	mkdir -p $(CACHE_DIR) $(CACHE_DIR)/options
-	[[ -d $(FUNCS_DIR) ]] \
-		&& echo 1 > $(CACHE_DIR)/got_func \
-		|| echo 0 > $(CACHE_DIR)/got_func
 
 $(FUNCS_DIR)/:
 	@$(info creating $(FUNCS_DIR)/ dir)
@@ -288,6 +230,59 @@ $(CACHE_DIR)/print_help.sh: $(CACHE_DIR)/help_table.txt $(CACHE_DIR)/synopsis.tx
 		cat $(CACHE_DIR)/help_table.txt
 		printf '%s\n' 'EOB' '}'
 	} > $@
+
+$(function_createconf): $(conf_files) | $(FUNCS_DIR)/
+	@$(info making $@)
+	{
+		printf '%s\n' \
+			'$(SHBANG)'                                                            \
+			''                                                                     \
+			'### _createconf() function is automatically generated'                \
+			'### from makefile based on the content of the $(CONF_DIR)/ directory' \
+			''
+
+
+		fstyle=$(FUNC_STYLE)
+		printf "_createconf$${fstyle}\n"
+
+		echo 'local trgdir="$$1"'
+
+		echo 'mkdir -p $(subst $(CONF_DIR),"$$trgdir",$(conf_dirs))'
+		for f in $(conf_files); do
+
+			echo ""
+			echo 'if [[ -d $$__dir ]]; then #bashbud'
+			echo "cat \"\$$__dir/$$f\" > \"$${f/$(subst /,\/,$(CONF_DIR))/\$$trgdir}\" #bashbud"
+			echo 'else #bashbud'
+			echo "cat << 'EOCONF' > \"$${f/$(subst /,\/,$(CONF_DIR))/\$$trgdir}\""
+			cat "$$f"
+			echo "EOCONF"
+			echo 'fi #bashbud'
+			[[ -x $$f ]] && echo "chmod +x \"$${f/$(subst /,\/,$(CONF_DIR))/\$$trgdir}\""
+		done
+
+		echo '}'
+	} > $@
+
+$(function_awklib): $(awk_files) | $(FUNCS_DIR)/
+	@$(info making $@)
+	{
+		printf '%s\n' \
+			'$(SHBANG)'                                                             \
+			''                                                                      \
+			'### _awklib() function is automatically generated'                     \
+			'### from makefile based on the content of the $(AWK_DIR)/ directory'   \
+			''
+
+		fstyle=$(FUNC_STYLE)
+		printf "_awklib$${fstyle}\n"
+		printf '%s\n' \
+			'[[ -d $$__dir ]] && { cat "$$__dir/$(AWK_DIR)/"* ; return ;} #bashbud' \
+			"cat << 'EOAWK'"
+		cat $(awk_files)
+		printf '%s\n' "EOAWK" '}'
+	} > $@
+
 $(CACHE_DIR)/options_in_use $(CACHE_DIR)/getopt &: $(OPTIONS_FILE) | $(CACHE_DIR)/
 	@$(info parsing $(OPTIONS_FILE))
 	mkdir -p $(DOCS_DIR)/options

--- a/src/i3Kornhe/config.mak
+++ b/src/i3Kornhe/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3Kornhe
 VERSION      := 0.669
 CREATED      := 2017-12-12
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := options

--- a/src/i3flip/config.mak
+++ b/src/i3flip/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3flip
 VERSION      := 0.101
 CREATED      := 2018-01-03
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := i3flip [--move|-m] DIRECTION

--- a/src/i3fyra/config.mak
+++ b/src/i3fyra/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3fyra
 VERSION      := 1.25
 CREATED      := 2017-01-14
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := options

--- a/src/i3get/config.mak
+++ b/src/i3get/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3get
 VERSION      := 0.8
 CREATED      := 2017-03-08
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := i3get [OPTIONS]

--- a/src/i3gw/config.mak
+++ b/src/i3gw/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3gw
 VERSION      := 0.999
 CREATED      := 2017-01-11
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := i3gw MARK

--- a/src/i3king/config.mak
+++ b/src/i3king/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3king
 VERSION      := 0.33
 CREATED      := 2021-06-01
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := i3king [OPTIONS]

--- a/src/i3list/config.mak
+++ b/src/i3list/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3list
 VERSION      := 0.51
 CREATED      := 2017-10-06
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := options

--- a/src/i3menu/config.mak
+++ b/src/i3menu/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3menu
 VERSION      := 0.11
 CREATED      := 2018-07-21
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := i3menu [OPTIONS]

--- a/src/i3run/config.mak
+++ b/src/i3run/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3run
 VERSION      := 0.2
 CREATED      := 2017-04-20
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := options

--- a/src/i3var/config.mak
+++ b/src/i3var/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3var
 VERSION      := 0.050
 CREATED      := 2017-12-22
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := options

--- a/src/i3viswiz/config.mak
+++ b/src/i3viswiz/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3viswiz
 VERSION      := 0.55
 CREATED      := 2018-01-18
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := options

--- a/src/i3zen/config.mak
+++ b/src/i3zen/config.mak
@@ -1,6 +1,7 @@
 NAME         := i3zen
 VERSION      := 0.1
 CREATED      := 2017-09-11
+UPDATED      := 2022-05-21
 AUTHOR       := budRich
 CONTACT      := https://github.com/budlabs/i3ass
 USAGE        := $(NAME) [OPTIONS]

